src/plugins/unified_search/public/query_string_input/query_bar_top_row.tsx

```diff
@@ -60,6 +60,9 @@ import type {
 } from '../typeahead/suggestions_component';
 import './query_bar.scss';
 
+const isMac = navigator.platform.toLowerCase().indexOf('mac') >= 0;
```

Comment:
This is clever. Feels like this detection should live somewhere central in Kibana and be available to plugins. Not really feedback for this PR.